integration_tests/mockApis/adjudications.ts

```diff
@@ -1002,6 +1002,21 @@ const stubGetDataInsightsChart = ({ id, response = {} }): SuperAgentRequest =>
     },
   })
 
+const stubCreatePunishmentComment = ({ adjudicationNumber, response }): SuperAgentRequest =>
```

Comment:
you need to add integration tests for amend a comment, and delete comment

Reply:
tests added